src/navigation/navigation.tsx

```diff
@@ -144,6 +155,41 @@ export const WalletNavigation = createStackNavigator(
         Validator: {
             screen: ValidatorScreen
         },
+        DelegateSelectValidator: {
```

Comment:
first screen from flows should be renamed. For this specific case: `PosDelegate`. Also all POS screen should be prefixed with `Pos`